code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -218,7 +218,6 @@
 		if(isSynthetic())
 			output += "Current Battery Charge: [nutrition]\n"
 
-		if(isSynthetic())
```

Comment:
Pain. Why did this part need that if(isSynthetic()) twice? Spaghetti bruh moment.